api_v3/services/BaseEntryService.php

```diff
@@ -153,6 +153,10 @@ protected function attachResource(kResource $resource, entry $dbEntry, asset $as
 				$service = new LiveStreamService();
     			$service->initService('liveStream', 'liveStream', $this->actionName);
     			break;
+		    case entryType::DOCUMENT:
+			    $service = new DocumentsService();
```

Comment:
this service is under plugin, we cant just assume it exists on all environments, need to check if the plugin for document is enabled before accessing it